Source/Core/VideoCommon/VertexShaderGen.cpp

```diff
@@ -23,7 +23,6 @@ VertexShaderUid GetVertexShaderUid()
   memset(uid_data, 0, sizeof(*uid_data));
 
   _assert_(bpmem.genMode.numtexgens == xfmem.numTexGen.numTexGens);
-  _assert_(bpmem.genMode.numcolchans == xfmem.numChan.numColorChans);
```

Comment:
Can the assert on line 25 be removed as well, or is that unaffected by this PR? That assertion causes a warning with SSX Tricky: [Assertion failure in VertexShaderGen.cpp: bpmem numtexgens doesn't equal xfmem numtexgens](https://bugs.dolphin-emu.org/issues/9930).